components/List/tests/snapshot/MDCSelfSizingStereoCellSnapshotTests.m

```diff
@@ -97,6 +97,8 @@ - (void)generateSnapshotAndVerifyForView:(UIView *)view {
   [self snapshotVerifyView:snapshotView];
 }
 
+// TODO(#7487): The size of the cell view sent for snapshot is not correct because
```

Comment:
Nit: Please use the URL of the Issue, not just the number.

Reply:
Addressed.